src/app/view/perspectives/post-it/document/post-it-document.component.scss

```diff
@@ -64,8 +64,15 @@
   }
 
   .attribute input {
-    font-weight: bold !important;
-    width: 80px;
+    width: 75px;
+    color: $primary;
+    font-style: italic;
+  }
+
+  .default-attribute input {
+    font-style: normal;
+    font-weight: bold;
+    color: #000000;
```

Comment:
I am not really sure if this is what @marvec meant. I think all attributes should be in italics and the default one in bold as well. But I might be wrong.

If I am right you don't really need to define some special class here and can just use:
```html
[class.font-weight-bold]="isDefaultAttribute(attributePair.attribute)"
```

Reply:
I've tried that. There was no way to tell which one is the default one, so I made it more visible.